src/controllers/accounts/accounts.ts

```diff
@@ -61,14 +70,16 @@ export class AccountsController extends EventEmitter {
 
     // eslint-disable-next-line @typescript-eslint/no-floating-promises
     this.initialLoadPromise = this.#load()
+    this.#authorizations = {}
   }
 
   async #load() {
     await this.#networks.initialLoadPromise
     await this.#providers.initialLoadPromise
-    const [accounts, accountPreferences] = await Promise.all([
+    const [accounts, accountPreferences, storageSignedMessages] = await Promise.all([
       this.#storage.get('accounts', []),
-      this.#storage.get('accountPreferences', undefined)
+      this.#storage.get('accountPreferences', undefined),
+      this.#storage.get('signedMessages', {})
```

Comment:
`signedMessages` should be read from `ActivityController` instead of directly accessing localStorage.

Reply:
However, I've seen that we need #authorizations only in the case we call `#updateAccountStates()`. If so, I would suggest to simply get it from ActivityController, in a similar way we do it here:

```
const entryPointAuthorizationMessageFromHistory = await this.activity.findMessage(
        account.addr,
        (message) =>
          message.fromActionId === ENTRY_POINT_AUTHORIZATION_REQUEST_ID &&
          message.networkId === meta.networkId
      )
```

Doing this, there is not need to read the signed messages from localStorage and keeping `#authorizations` variable up to date.